migrations/migrations/0002_archive_meetings.py

```diff
@@ -56,6 +56,7 @@ def position_init(self) -> None:
     def migrate_event(
         self,
         event: BaseEvent,
+        adsany: str,
```

Comment:
If I go into the dev shell (`make run-bash`) in the migration folder I get this result:
```
finn@4721ece4ff20:/app$ mypy .
Success: no issues found in 5 source files
finn@4721ece4ff20:/app$ mypy .
migrations/0002_archive_meetings.py:56: error: Signature of "migrate_event" incompatible with supertype "BaseMigration"
Found 1 error in 1 file (checked 5 source files)
```

(first call without this line, second call with this change). Can you reproduce this in the dev shell?